samples/scheduling/timer/Core_8/Sample/Program.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+partial class Program
+{
+    static ILog log = LogManager.GetLogger<Program>();
+
+    static async Task Main()
+    {
+        Console.Title = "Samples.Scheduling.Timer";
+        var endpointConfiguration = new EndpointConfiguration("Samples.Scheduling.Timer");
+        endpointConfiguration.UsePersistence<LearningPersistence>();
+        endpointConfiguration.UseTransport<LearningTransport>();
+
+        var endpointInstance = await Endpoint.Start(endpointConfiguration)
+            .ConfigureAwait(false);
+
+        #region ScheduleUsingTimer
+        var interval = TimeSpan.FromSeconds(5);
+
+        var timer = new Timer(async state =>
+        {
+            await endpointInstance.SendLocal(new MyScheduledTask());
```

Comment:
What happens that send fails? I think that would bring down the process.

Reply:
It does, I have now added a try catch